pages/FAQ.md

```diff
@@ -0,0 +1,38 @@
+# FAQ
+
+## I have an error/bug with Slimefun
+Ok, first things first, join our [Support Server](https://slimefun.dev/discord) if you aren't in it already and go to #bug-discussions. We need to determine if it is a bug, user error or intended behaviour.
+
+Now, send us the following information (in the #bug-discussions channel):
+1. Run /sf versions and send us a screenshot. We need the exact versions you are using, otherwise, we will not be able to help you.
+"latest" is not helping us at all. So please run that command. (If you don't have access to this command then Shift-Right-Click your Slimefun Guide, you can find your versions in the uppermost middle slot.)
+
+2. Check your console, are there any errors? (If so, then post them via https://pastebin.com/)
+
+3. What do you intend to happen and what is happening?
+
+Once you've sent us all this info then one of our staff or a community member will help you. **Do not ping any role or members**. If some time has passed (15+ mins) then you may ping the helpful role (`@Helpful`). Being impatient and pinging roles/members may get you warned or kicked.
+
+## What's the difference between the talisman and ender talisman?
+The Ender Talismans work even while in the Ender Chest however the normal [Talismans](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Talismans) requires for it to be in your inventory.
+
+## What can I do with stone chunks?
+4 Stone Chunks can be put into the [Compressor](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Compressor)
+
+## How do I disable items per-world?
+You can disable items per-world in the file `/plugins/Slimefun/whitelist.yml` by finding the item ID and setting it to false.
```

Comment:
That path is no longer correct, it was moved to `/plugins/Slimefun/world-settings/` where there is a seperate .yml file for each world because that one single file had up to 60K lines.., You can also disable items per addon now.